api/src/main/resources/assets/index.html

```diff
@@ -5635,6 +5635,7 @@
 	return {$: 'Set_elm_builtin', a: a};
 };
 var $elm$core$Set$empty = $elm$core$Set$Set_elm_builtin($elm$core$Dict$empty);
+var $author$project$Main$emptyFormValues = {executionName: $elm$core$Maybe$Nothing, fromSchema: $elm$core$Dict$empty};
 var $elm$browser$Browser$Navigation$pushUrl = _Browser_pushUrl;
```

Comment:
Is this file supposed to be checked in?

Reply:
I think yes? I decided to check it in because of the built website being checked in. I don't have a strong opinion.